docs/api/dbus/meson.build

```diff
@@ -0,0 +1,25 @@
+spec_to_docbook = find_program('spec-to-docbook.sh')
+
+packagekit_ref_xml = custom_target(
+	'org.freedesktop.PackageKit.ref.xml',
+	input: join_paths(meson.source_root(), 'src', 'org.freedesktop.PackageKit.xml'),
+	output: 'org.freedesktop.PackageKit.ref.xml',
+	command: [
+		spec_to_docbook,
+		files('spec-to-docbook.xsl'),
+		'@INPUT@',
+		'@OUTPUT@',
+	],
```

Comment:
Adding `build_by_default: true` to these custom targets will allow `ninja dist` to work.